### PR TITLE
Revisar traducción en español.

### PR DESCRIPTION
**Pull request by ghost**
Se ha mejorado la presentación (hasta donde he podido), corregido faltas de ortografía, gramática y se han añadido nexos en algunas ubicaciones del texto.

### CODE REVIEW

**Comment by alexandrumuntenas** [comment]
> Se ha mejorado la presentación (hasta donde he podido), corregido faltas de ortografía, gramática y se han añadido nexos en algunas ubicaciones del texto.

The presentation has been improved (as far as I could), spelling and grammar mistakes have been corrected, and links have been added in some places in the text.

File: translations/README-es.md [hunk 1]
```diff
@@ -512,203 +518,202 @@ Puede ver mi código aquí:
...
+    - [ ] [Using Stacks Last-In First-Out (video)](https://archive.org/details/0102WhatYouShouldKnow/05_01-usingStacksForLast-inFirst-out.mp4).
     - [ ] No implementaré. Implementando con arreglos es trivial.
 
 - ### Cola o fila
-    - [ ] [Using Queues First-In First-Out(video)](https://archive.org/details/0102WhatYouShouldKnow/05_03-usingQueuesForFirst-inFirst-out.mp4)
-    - [ ] [Queue (video)](https://www.coursera.org/learn/data-structures/lecture/EShpq/queue)
-    - [ ] [Circular buffer/FIFO](https://en.wikipedia.org/wiki/Circular_buffer)
-    - [ ] [Priority Queues (video)](https://archive.org/details/0102WhatYouShouldKnow/05_04-priorityQueuesAndDeques.mp4)
+    - [ ] [Using Queues First-In First-Out(video)](https://archive.org/details/0102WhatYouShouldKnow/05_03-usingQueuesForFirst-inFirst-out.mp4).
+    - [ ] [Queue (video)](https://www.coursera.org/learn/data-structures/lecture/EShpq/queue).
```
Comment (jwasham) [inline translations/README-es.md:636]: There are a lot of lines like this where only a period was added. Please remove the extra period from those lines to make it easier to see the differences.

Comment (alexandrumuntenas) [reply translations/README-es.md]: Just removed most of the periods added... I just reviewed the punctuation rules and noticed that in listings in which long sentences are not included, periods should not be added at the end. 

Let me know what you think.

**Comment by jwasham** [comment]
Thank you very much!

**Comment by backeyshy** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Pull request merged**